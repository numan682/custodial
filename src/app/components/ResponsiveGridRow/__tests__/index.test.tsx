import * as React from 'react'
import { render } from '@testing-library/react'

import { ResponsiveGridRow } from '..'

describe('<ResponsiveGridRow />', () => {
  it('should render component', () => {
    const { container } = render(<ResponsiveGridRow label="Type" value="transfer" />)

    expect(container).toMatchSnapshot()
  })
})
