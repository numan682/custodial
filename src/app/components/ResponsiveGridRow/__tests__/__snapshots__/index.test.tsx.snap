// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ResponsiveGridRow /> should render component 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  max-width: 100%;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  max-width: 100%;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

.c1 {
  font-size: 18px;
  line-height: 24px;
  font-weight: bold;
}

<div>
  <div
    class="c0"
  >
    <span
      class="c1"
    >
      Type
    </span>
  </div>
  <div
    class="c2"
  >
    transfer
  </div>
</div>
`;
